kotlin/map-rotation/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.esri.arcgisruntime.sample.maprotation.MainActivity">
+
+    <!-- MapView -->
+    <com.esri.arcgisruntime.mapping.view.MapView
+        android:id="@+id/mapView"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginBottom="0dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0" />
+
+    <SeekBar
+        android:id="@+id/rotationSeekBar"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginBottom="68dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_toStartOf="@+id/rotationValueText"
+        android:max="360"
+        app:layout_constraintBottom_toBottomOf="@+id/mapView"
+        app:layout_constraintLeft_toLeftOf="@+id/mapView"
+        app:layout_constraintRight_toLeftOf="@+id/rotationValueText"
+        android:layout_toLeftOf="@+id/rotationValueText" />
+
+    <TextView
+        android:id="@+id/rotationValueText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignTop="@+id/rotationSeekBar"
+        android:layout_marginEnd="16dp"
+        android:text="@string/initial_rotation_text"
+        android:textSize="18sp"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="@+id/rotationSeekBar"
+        app:layout_constraintRight_toRightOf="parent"
+        tools:text="180"
+        android:layout_alignParentRight="true"
+        android:layout_marginRight="16dp" />
```

Comment:
these are probably left over from the old sample and aren't needed anymore

Reply:
you're right and i've removed them, but I only ever had them because android studio was yelling at me to add them, but now it's not